app/scripts/modules/google/src/loadBalancer/configure/tcp/listener.html

```diff
@@ -0,0 +1,28 @@
+<div class="container-fluid form-horizontal">
+  <ng-form name="listenerForm">
+    <div class="form-group">
+      <div class="col-md-9 col-md-offset-2">
+        <table class="table table-condensed packed" style="table-layout: fixed; width: 390px;">
+          <thead>
+          <tr>
+            <th>Port</th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr>
+            <td>
+              <input class="form-control input-sm"
+                     readonly
```

Comment:
This should actually be a dropdown list, for both SSL and TCP Proxy. Will change